src/lib/i18n/types.ts

```
@@ -0,0 +1,16 @@
+import enMessages from "@/lib/i18n/translations/en.json";
```

ESLint

[error] There should be no empty line within import group

(import/order)